src/tree/private_tree.rs

```diff
@@ -0,0 +1,209 @@
+//! A data structure holding information about the leaf node in the tree that
+//! belongs to the current client.
+//!
+
+use super::{index::NodeIndex, path_keys::PathKeys, TreeError};
+use crate::ciphersuite::{Ciphersuite, HPKEKeyPair, HPKEPrivateKey, HPKEPublicKey};
+use crate::codec::{Codec, CodecError};
+use crate::messages::CommitSecret;
+use crate::schedule::hkdf_expand_label;
+
+#[derive(Debug)]
+pub(crate) struct OwnLeaf {
```

Comment:
Since you already introduce the term private tree, shouldn't this be renamed to `PrivateTree`?

Reply:
yeah... I meant to do that ...